[Question] parallel_api_test

### Question

Hi, I want to ask that line 120 of parallel_api_test

I got error of `AssertionError: ['agent_0', 'agent_1', 'agent_2'] != set()`
I found it occurs when my custom env is done!
![image](https://user-images.githubusercontent.com/49419937/215052632-015d5771-6ed0-4610-bb05-5b4dc8a75a0a.png)


```
f"{par_env.agents} != {live_agents}"
```

Why should "par_env.agents == live_agents"? and how can I fix it?

thanks!

--- a/tutorials/EnvironmentCreation/2-AddingGameLogic.py
+++ b/tutorials/EnvironmentCreation/2-AddingGameLogic.py
@@ -81,6 +81,7 @@ class CustomEnvironment(ParallelEnv):
         if self.timestep > 100:
             rewards = {"prisoner": 0, "guard": 0}
             truncations = {"prisoner": True, "guard": True}
+            self.agents = []
         self.timestep += 1
 
         # Get observations

